build/src/wrapper.c

```diff
@@ -311,3 +311,12 @@ int KEEPALIVE column_bytes(int entry_id, int stmt_id, int col) {
   }
   return sqlite3_column_bytes(stmt, col);
 }
+
+const char* KEEPALIVE column_name(int entry_id, int stmt_id, int col) {
```

Comment:
This is very good! Could you add two more wrappers for `sqlite3_column_origin_name` and `sqlite3_column_table_name` in the same manner as this one?

(And make sure to re-base on master before you compile, as there is now support for automatically updating the `sqlite.js` file as part of the `make` invocation 😉)